plugins/inputs/logfile/fileconfig.go

```diff
@@ -211,8 +211,9 @@ func (config *FileConfig) timestampFromLogLine(logValue string) (time.Time, stri
 			}
 		}
 		if config.TrimTimestamp {
-			// Trim the entire timestamp portion (from start to end of the match)
-			return timestamp, logValue[:index[0]] + logValue[index[1]:]
+			// Trim the entire timestamp portion and leading whitespaces
+			// The whitespace characters being removed are: space, tab, newline, and carriage return ( " \t\n\r")
+			return timestamp, strings.TrimLeft(logValue[:index[0]]+logValue[index[1]:], " \t\n\r")
```

Comment:
Makes sense to me. Same concept in OTEL https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/765ed69d45e1220dfedb7ebdad8df5d09c9e73a5/pkg/stanza/trim/trim.go#L49